### ISSUE TITLE
Help with calculating mAP for a test set

### ISSUE THREAD

**Issue by solder-fumes-asthma** [google/automl#968]
Hello. I have been struggling with finding the mAP and FPS for a test set.

I have a series of images with labels that I have converted to .tfrecord, I have one for each split: train, valid and test. I have successfully trained a model until it has overfitted, so in my _model_output/_ folder I have some overfitted _.ckpt_ files and inside _model_output/backup/_ I have the checkpoint files for my best model on validation set. 

During training, AP metrics for the validation set were calculated each iteration. I now need the same information for my test set of images on the final model. 

Please can someone tell me how can I get it? I have tried a lot of commands that i have found in this repo and some external places but none have worked so far. Starting to get a bit desperate.

Thanks a lot and best regards.
